CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ This is a changelog for Piwik platform developers. All changes for our HTTP API'
 ### Library updates
 * Updated pChart library from 2.1.3 to 2.1.4. The files were moved from the directory `libs/pChart2.1.3` to `libs/pChart`
 
+### Other
+* During bulk tracking, if a request references a non-existant website, it will be skipped. So bulk tracking requests with one or more of these will succeed where they would have failed before. The number of skipped requests is now reported in bulk tracking responses.
```

Comment:
Should we maybe list it as "breaking change"? It doesn't break code but in terms of functionality? "Breaking change" is maybe too negative though since it is rather a fix. Actually: I'm not sure if we need to list it here at all. It's not really code / platform related. This is more an important notice in the regular changelog.